src/App.js

```diff
@@ -9,7 +10,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div>{Article(...Object.values(article))}</div>
```

Comment:
```suggestion
  <Article {...article} />
```
Компоненты вызываются как self-closing tags, а не как функции